mythril/laser/ethereum/call.py

```diff
@@ -178,14 +178,6 @@ def get_call_data(
             else memory_size
         ),
     )
-
```

Comment:
I'll just trust you that this fixes everything :) Can't replicate it right now because I'd have to context switch my whole test environment

Reply:
I am not sure yet. I didn't run it with the contract on atlassian. This is a bugfix for a tailor made contract which is a bit similar to original contract. But it should atleast fix one problem. Will test it then